crates/wat/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wat"
-version = "1.0.28"
+version = "1.1.0"
```

Comment:
Oh FWIW this only changes in minor ways so I've been keeping it at 1.0.x instead of 1.1.0